app/components/primer/beta/auto_complete.rb

```diff
@@ -42,7 +42,16 @@ class AutoComplete < Primer::Component
 
       # Optional icon to be rendered before the input. Has the same arguments as <%= link_to_component(Primer::OcticonComponent) %>.
       #
-      renders_one :icon, Primer::OcticonComponent
+      renders_one :icon, lambda { |**system_arguments|
+        aria_label = aria("label", system_arguments) || @aria_label
+        if aria_label.present?
+          system_arguments[:"aria-label"] = aria_label
```

Comment:
Note the addition here, *without* a subtraction of the `input` field's `aria-label`. It sounds okay to me with voiceover, not sure how important it is to have just one aria-label.

Reply:
I'm interested to hear feedback from @jscholes on labelling this the same as the input.

Reply:
We spoke and he recommends changing the overall API to only allow for
- `label`
- `icon`
- `isLabelVisible`

in the interest of not allowing for a conflict between the `label` and if someone has provided an `aria-label`. But we'll be forking that work to be done elsewhere. I'll create an issue as follow-up.